content/zh/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -79,9 +78,69 @@ spec:
     $ kubectl -n istio-system apply -f attribute_gen_service.yaml
     {{< /text >}}
 
-1. 更改生效后，访问 Prometheus 并查找新的或更改的维度，例如 `reviews` Pod 中的 `istio_requests_total`。
+1. 使用以下命令从 `istio-system` 命名空间中查找
+   `stats-filter-{{< istio_version >}}` `EnvoyFilter` 资源：
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter | grep ^stats-filter-{{< istio_version >}}
+    stats-filter-{{< istio_version >}}                    2d
+    {{< /text >}}
+
+1. 使用以下命令创建 `EnvoyFilter` 配置的本地文件系统副本：
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter stats-filter-{{< istio_version >}} -o yaml > stats-filter-{{< istio_version >}}.yaml
+    {{< /text >}}
+
+1. 使用文本编辑器打开 `stats-filter-{{< istio_version >}}.yaml`
+   并找到 `name: istio.stats` 扩展配置。对其更新以将
+   `requests_total` 标准指标中的 `request_operation`
+   维度映射到 `istio_operationId` 属性。更新后的配置文件部分应如下所示。
+
+    {{< text json >}}
+        name: istio.stats
+        typed_config:
+          '@type': type.googleapis.com/udpa.type.v1.TypedStruct
+          type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+          value:
+            config:
+              configuration:
+                "@type": type.googleapis.com/google.protobuf.StringValue
+                value: |
+                  {
+                    "metrics": [
+                     {
+                       "name": "requests_total",
+                       "dimensions": {
+                         "request_operation": "istio_operationId"
+                       }
+                     }]
+                  }
+    {{< /text >}}
+
+1. 保存 `stats-filter-{{< istio_version >}}.yaml`，然后使用以下命令应用配置：
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f stats-filter-{{< istio_version >}}.yaml
+    {{< /text >}}
+
+1. 将以下配置添加到网格配置中。这导致将 `request_operation`
+   添加为 `istio_requests_total` 指标的新维度。如果没有它，则会创建一个名为
+   `envoy_request_operation___somevalue___istio_requests_total` 的新指标。
+
+    {{< text yaml >}}
+    meshConfig:
+      defaultConfig:
+        extraStatTags:
+        - request_operation
+    {{< /text >}}
+
+1. 通过向您的应用程序发送流量来生成指标。
+
+1. 更改生效后，访问 Prometheus 并查找新的或更改的维度，
+   例如 `istio_requests_total`。
```

Comment:
Need to confirm. Is there anything wrong? The zh text is inconsistent with en upstream:

https://github.com/istio/istio.io/blob/master/content/en/docs/tasks/observability/metrics/classify-metrics/index.md